tests/unit/libexpr/main.cc

```diff
@@ -34,6 +34,9 @@ int main (int argc, char **argv) {
     setEnv("_NIX_TEST_NO_SANDBOX", "1");
     #endif
 
+    // For pipe operator tests in trivial.cc
+    experimentalFeatureSettings.set("experimental-features", "pipe-operators");
+
```

Comment:
So very sorry for doing this here; couldn't see a way to enable this just for the `TrivialExpressionTest` fixture. I saw other fixtures creating `mockXpSettings`, but they seem to be exercising API that accepts a settings object, and `EvalState::parseExprFromString` does not as far as I can tell. I didn't want to make this change bigger by hacking up the `EvalState` interface and threading a settings object all the way to the lexer... but maybe you think I ought to?

Reply:
This flag doesn't affect other functionality, so this won't be a problem for the unit tests.

The flag behavior is tested with the functional tests.